src/client/components/hazard-list/hazard-list.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <ul class="hazard-list">
+    <li
+      v-for="(item, index) in list"
+      :key="index"
+      class="hazard-list__list-item md-body"
+    >
+      <md-radio
+        v-model="selectedItem"
+        :value="item"
```

Comment:
input is probably going to be an object, so this could be `item.title` for example